lib/LedgerSMB/Template.pm

```diff
@@ -270,26 +275,105 @@ sub _valid_language {
 }
 
 sub _preprocess {
-    my ($self, $vars) = @_;
-    return unless $self->{myconfig};
-    use LedgerSMB;
-    my $type = ref($vars);
+    my ($rawvars, $escape) = @_;
+    return undef unless defined $rawvars;
 
-    if ($type eq 'SCALAR' || !$type){
-        return;
-    }
-    if ($type eq 'ARRAY'){
-        for (@$vars){
-            if (ref($_)){
-                $self->_preprocess($_);
-            }
+    { # pre-5.14 compatibility block
+      # Really? -YL
```

Comment:
Yup. Really. Now, if you wonder what problem is being solved, there's a better palce to discuss that than in the code 😊.

The problem worked around btw, is with the localized reference to $@.